sdk/appconfiguration/app-configuration/test/public/featureFlag.spec.ts

```diff
@@ -192,4 +187,21 @@ describe("AppConfigurationClient - FeatureFlag", () => {
       await client.deleteConfigurationSetting({ key: secondSetting.key });
     });
   });
+
+  describe("FeatureFlag utils", () => {
```

Comment:
Future PR: probably want to do the same kind of unit-test split we did in Service Bus so it's easier to pick apart the live tests from the unit tests.

Reply:
I did think about it while writing these tests... but..
I think this is fine since there is no need to split unit tests because all the tests run as unit tests in playback mode.